include/odp/api/spec/packet.h

```diff
@@ -2150,6 +2150,42 @@ uint32_t odp_packet_payload_offset(odp_packet_t pkt);
  */
 int odp_packet_payload_offset_set(odp_packet_t pkt, uint32_t offset);
 
+/** Packet Tx completion mode */
+typedef enum odp_packet_tx_compl_mode_t {
+	/** Packet Tx completion event is disabled. */
+	ODP_PACKET_TX_COMPL_DISABLED,
```

Comment:
This could mention that when mode is ODP_PACKET_TX_COMPL_DISABLED, all other odp_packet_tx_compl_opt_t fields are ignored (user do not need to set e.g. queue = QUEUE_INVALID).